docs/QuickStart.md

```diff
@@ -0,0 +1,58 @@
+## Getting Started / Quick Start
+
+### Features
+
+* Parses Mustache templates and an extended template format (in YAML)
+* Generates a view schema from parsed template data
+* Supports Mustache partials and sections
+* Renders templates with user-provided views
+* Validates user-provided views against generated view schema
+* Includes a [command line interface](#cli)
+
+### CLI
```

Comment:
Basic CLI usage may be better suited for the README

Reply:
Would the link to the CLI section of the Quick Start be sufficient on the README, 
or do we need to either duplicate this info on both pages or 
remove the CLI section from the Quick Start and only have it on the README?